src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.targets

```diff
@@ -166,7 +186,7 @@
 
     <!-- Copy the Razor debug information file (.pdb), if any -->
     <Copy
-      SourceFiles="$(_RazorDebugSymbolsIntermediatePath)"
+      SourceFiles="@(_RazorDebugSymbolsIntermediatePath)"
```

Comment:
This was an oopsie. Not sure why it's not breaking a test.

Reply:
`DestinationFiles` in this target also looks a bit supsect. I'll have a look

Reply:
I'll take a look at it.